conans/test/unittests/model/editable_cpp_info/work_on_items_test.py

```diff
@@ -1,80 +1,32 @@
-                      EditableCppInfo._work_on_item("D:\\Windows-double-slash\\include",
-                                                    base_path, None, None))
-
-    @parameterized.expand([(False,), (True,)])
-    def test_unix(self, make_abs):
-        base_path = os.path.dirname(__file__) if make_abs else None
-
-        self.assertIn(os.path.join(os.sep, 'abs', 'path', 'include'),
-                      EditableCppInfo._work_on_item("/abs/path/include", base_path, None, None))
-
-    @parameterized.expand([(False,), (True,)])
-    def test_placeholders(self, make_abs):
-        base_path = os.path.dirname(__file__) if make_abs else None
+        self.assertEqual("", EditableCppInfo._work_on_item("", None, None))
 
+    def test_placeholders(self):
         settings = Settings.loads(default_settings_yml)
         settings.compiler = 'Visual Studio'
         settings.compiler.version = '14'
         settings.build_type = 'Debug'
 
-        self.assertIn(_make_abs(base_path, 'src', 'Visual Studio14', 'Debug', 'include'),
-                      EditableCppInfo._work_on_item("src/{settings.compiler}{settings.compiler.version}/{settings.build_type}/include",
-                                                    base_path=base_path, settings=settings,
-                                                    options=None))
-        self.assertIn(os.path.join('C:' + os.sep, 'Visual Studio', 'include'),
-                      EditableCppInfo._work_on_item("C:\\{settings.compiler}\\include\\",
-                                                    base_path=base_path, settings=settings,
-                                                    options=None))
-        self.assertIn(os.path.join(os.sep, 'usr', 'path with spaces', 'Visual Studio', 'dir'),
-                      EditableCppInfo._work_on_item("/usr/path with spaces/{settings.compiler}/dir",
-                                                    base_path=base_path, settings=settings,
-                                                    options=None))
+        self.assertEqual('src/Visual Studio14/Debug/include',
+                         EditableCppInfo._work_on_item("src/{settings.compiler}{settings.compiler.version}/{settings.build_type}/include",
+                                                       settings=settings,
+                                                       options=None))
+        self.assertEqual('C:/Visual Studio/include/',
+                         EditableCppInfo._work_on_item("C:\\{settings.compiler}\\include\\",
+                                                       settings=settings,
+                                                       options=None))
+        self.assertEqual('/usr/path with spaces/Visual Studio/dir',
+                         EditableCppInfo._work_on_item("/usr/path with spaces/{settings.compiler}/dir",
+                                                       settings=settings,
+                                                       options=None))
```

Comment:
More tests:
 * windows-like with just one backward slash
 * path with an ending slash -> I think that this ending slash should be removed (or added one to every other path)